interface/billing/sl_receipts_report.php

```diff
@@ -716,9 +716,11 @@ function sel_diagnosis() {
                 </td>
                         <?php if ($form_procedures) { ?>
                 <td align="right">
-                            <?php echo text(FormatMoney::getBucks($doctotal2)) ?>
```

Comment:
maybe this should be
                            <?php echo text(FormatMoney::getBucks($doctotal2, true)) ?>
to return 0.00?